docs/cli.md

```diff
@@ -523,28 +523,73 @@ stats_tests.cpp
 ```
 
 ### Input redirection `<`
-Input redirection sends the contents of a file to the input of a command.  It's useful for automating input.  Here's an example from EECS 280 project 1 ([full example](https://eecs280staff.github.io/p1-stats/#testing-1)).
+*Input redirection* sends the contents of a file to the input of a program.   Input redirection is useful for automating program input.
 
-Put the input that you would type in a file.
+Here's an example program.
+```c++
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main() {
+  cout << "What's your name?" << endl;
+  string name;
+  cin >> name;
+  cout << "Hello " << name << "!\n";
+}
+```
+
+Without input redirection, the user types input (highlighted).
+```console
+$ g++ main.cpp -o main.exe
+$ ./main.exe
+What's your name?
+Drew
```

Comment:
```suggestion
Drew       # user input
```
Not sure if this is necessary, but might be nice to explicitly show which part is user input and which is program output.

Reply:
I'm a little worried about the reader thinking that `# user input` is part of the actual user input.